### PR TITLE
Merge PR #167

### PR DESCRIPTION
**Pull request by MarkDaoust**
#167

### CODE REVIEW

File: site/en/tutorials/images/_image_classigication.ipynb [hunk 1]
```diff
@@ -0,0 +1,1459 @@
...
+    {
+      "metadata": {
+        "colab_type": "text",
+        "id": "KwQtSOz0VrVX"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"#\"><img src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" />View on TensorFlow.org</a>\n",
```
Comment (lamberta) [inline site/en/tutorials/images/_image_classigication.ipynb]: Add these links

File: site/en/tutorials/images/_image_classigication.ipynb [hunk 2]
```diff
@@ -0,0 +1,1459 @@
...
+      ]
+    },
+    {
+      "metadata": {
+        "colab_type": "text",
+        "id": "gN7G9GFmVrVY"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "In this tutorial, we will discuss how to classify cats vs dogs from images. We'll build an image classifier using **tf.keras**'s **Sequential** model and load data using **ImageDataGenerator**.\n",
```
Comment (lamberta) [inline site/en/tutorials/images/_image_classigication.ipynb]:
```suggestion
        "In this tutorial, we will discuss how to classify cats vs dogs from images. We'll build an image classifier using a `tf.keras.models.Sequential` model and load data using `ImageDataGenerator`.\n",
```

File: site/en/tutorials/images/_image_classigication.ipynb [hunk 3]
```diff
@@ -0,0 +1,1459 @@
...
+        "\n",
+        "## Specific concepts that will be covered:\n",
+        "In the process, we will build practical experience and develop intuition around the following concepts\n",
+        "* **Building data input pipelines using the tf.keras's ImageDataGenerator Class** - how can we efficiently work with data on disk to interface with our model? \n",
+        "* **Overfitting** - what is it, how to identify it, and how can we prevent it? \n",
+        "* **Data Augmentation** and **Dropout** -  Key techniques to fight overfitting in computer vision tasks that we will incorporate into our data pipeline and image classifier model. \n",
```
Comment (lamberta) [inline site/en/tutorials/images/_image_classigication.ipynb]:
```suggestion
        "* _Data augmentation_ and _dropout_ —Key techniques to fight overfitting in computer vision tasks that we will incorporate into our data pipeline and image classifier model. \n",
```

File: site/en/tutorials/images/_image_classigication.ipynb [hunk 4]
```diff
@@ -0,0 +1,1459 @@
...
+        "id": "gN7G9GFmVrVY"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "In this tutorial, we will discuss how to classify cats vs dogs from images. We'll build an image classifier using **tf.keras**'s **Sequential** model and load data using **ImageDataGenerator**.\n",
+        "\n",
+        "## Specific concepts that will be covered:\n",
+        "In the process, we will build practical experience and develop intuition around the following concepts\n",
+        "* **Building data input pipelines using the tf.keras's ImageDataGenerator Class** - how can we efficiently work with data on disk to interface with our model? \n",
+        "* **Overfitting** - what is it, how to identify it, and how can we prevent it? \n",
```
Comment (lamberta) [inline site/en/tutorials/images/_image_classigication.ipynb]:
```suggestion
        "* _Overfitting_ –What is it, how to identify it, and how can we prevent it?\n",
```

File: site/en/tutorials/images/_image_classigication.ipynb [hunk 5]
```diff
@@ -0,0 +1,1459 @@
...
+        "colab_type": "text",
+        "id": "gN7G9GFmVrVY"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "In this tutorial, we will discuss how to classify cats vs dogs from images. We'll build an image classifier using **tf.keras**'s **Sequential** model and load data using **ImageDataGenerator**.\n",
+        "\n",
+        "## Specific concepts that will be covered:\n",
+        "In the process, we will build practical experience and develop intuition around the following concepts\n",
+        "* **Building data input pipelines using the tf.keras's ImageDataGenerator Class** - how can we efficiently work with data on disk to interface with our model? \n",
```
Comment (lamberta) [inline site/en/tutorials/images/_image_classigication.ipynb]:
```suggestion
        "* Building *data input pipelines* using the `ImageDataGenerator` —How can we efficiently work with data on disk to interface with our model? \n",
```

File: site/en/tutorials/images/_image_classigication.ipynb (hunk 1 above)
Comment (MarkDaoust) [reply site/en/tutorials/images/_image_classigication.ipynb]: done

**Comment by MarkDaoust** [comment]
There's still more work I'd like to do, but this is a step in the right direction.

**Pull request merged**